iOS/WAStickersThirdParty/StickerCell.swift

```diff
@@ -12,26 +12,33 @@ final class StickerCell: UICollectionViewCell {
 
     private let imageView: UIImageView
 
-    var sticker: Sticker? {
-        willSet {
-            if newValue !== sticker {
-                imageView.image = nil
-            }
-        }
+    var sticker: WASticker? {
         didSet {
-            if oldValue !== sticker {
-                if let currentSticker = sticker {
-                    StickerPackManager.queue.async {
-                        let image = currentSticker.imageData.image(withSize: CGSize(width: 256, height: 256))
-                        DispatchQueue.main.async {
-                            if let sticker = self.sticker, currentSticker === sticker {
-                                self.imageView.image = image
-                            }
-                        }
-                    }
-                }
+            if let imageFile = sticker?.imageFile {
+                self.imageView.image = UIImage(named: imageFile)
             }
+
         }
+
+//        willSet {
```

Comment:
Again, if this is going to be merged, it should at least support both. Maybe add an if-statement checking if it is a sticker from the web or from the app bundle.